### ISSUE TITLE
Images not following global figure alignment unless has caption

### ISSUE THREAD

**Issue by victorfeagins** [quarto-dev/quarto-cli#4415]
### Bug description

I am using quarto version 1.2.269 on Windows 10 
RStudio 2022.12.0+353 "Elsbeth Geranium" Release (7d165dcfc1b6d300eb247738db2c7076234f6ef0, 2022-12-03) for Windows
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) RStudio/2022.12.0+353 Chrome/102.0.5005.167 Electron/19.1.3 Safari/537.36

When I render this document the first image is not centered. The middle image is centered and given a caption. The third image does not have a caption is centered and told explicitly to be centered.

```
---
title: "test"
author: "Victor"
format: 
  html:
    fig-align: center
---

![](https://www.clker.com/cliparts/1/9/e/4/13140637591549686593blue%20square-hi.png)

![Block](https://www.clker.com/cliparts/1/9/e/4/13140637591549686593blue%20square-hi.png)


![](https://www.clker.com/cliparts/1/9/e/4/13140637591549686593blue%20square-hi.png){fig-align="center"}
```

### Checklist

- [X] Please include a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.
- [X] Please [format your issue](https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us) so it is easier for us to read the bug report.
- [X] Please document the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?
- [X] Please document the operating system you're running. If on Linux, please provide the specific distribution.

**Comment by mcanouil** [comment]
For reference, this seems to be related to <https://github.com/quarto-dev/quarto-cli/issues/4315#issuecomment-1426023371>.

**Comment by cscheid** [comment]
Yes, I think it's a duplicate, but I'm keeping it open anyway to track the slightly different reprex.

**Comment by cscheid** [comment]
We have a simple fix, but we're very much on the fence on whether we _should_ do it.

The issue is that an image, by itself, isn't a _figure_. We've made similar fixes in other places (such as computational content) where most images are semantically "figures". But an image in a markdown document can have a number of different meanings that aren't "figure", and that wouldn't want to take a global `fig-align` setting.

Next year we plan to spend a good amount of effort making our document settings more uniform, and that might give us insight on where to land here. I'm going to push this to 1.5.
